### PR TITLE
Add slurm demo scripts

### LINKED ISSUE #5: Incorporate scripts from the `slurm-demo` repo.
In the interest of reducing the number of repos.

### PR DESCRIPTION
**Pull request by adamltyson**
Closes #5

### CODE REVIEW

**Comment by niksirbi** [comment]
I have hard time understanding what your intention is here, I'm not so experienced with submodules.
Is you plan to keep the demo script in the separate (soon-to-be-archived) repo? Can one even "submodule" an archived repo?

**Comment by adamltyson** [comment]
> I have hard time understanding what your intention is here, I'm not so experienced with submodules.

That's because I've messed up. Bear with me!

**Comment by adamltyson** [comment]
@niksirbi it should make more sense now.

File: index.qmd [hunk 1]
```diff
@@ -258,7 +258,7 @@ Always start a job (interactive or batch) before doing anything intensive to spa
 Clone a test script
 ```bash
 cd ~/
-git clone https://github.com/neuroinformatics-unit/slurm-demo
+git clone https://github.com/neuroinformatics-unit/course-behavioural-analysis-2023
```
Comment (niksirbi) [inline index.qmd:261]: shouldn't this be?

```suggestion
git clone https://github.com/neuroinformatics-unit/course-software-skills-hpc
```

File: demo/env.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: slurm_demo
```
Comment (niksirbi) [inline demo/env.yml:1]: I'm getting a warning that you should list pip itself as a conda dependency

**Review by niksirbi — APPROVED** [review]
Other than the one lingering comment about the `pip` dependency, LGTM

**Pull request merged**